packages/office-ui-fabric-react/src/components/FloatingPicker/BaseFloatingPicker.types.ts

```diff
@@ -84,14 +83,21 @@ export interface IBaseFloatingPickerProps<T> extends React.ClassAttributes<any>
    * ClassName for the picker.
    */
   className?: string;
+  /**
+   * How to render the Suggestions control used by this floating picker.
+   */
+  onRenderSuggestionControl?: React.ComponentType<BaseFloatingPickerSuggestionProps<T>>;
   /**
    * The properties that will get passed to the Suggestions component.
+   *
+   * @deprecated prefer binding properties on onRendersuggestionControl.
    */
-  pickerSuggestionsProps?: IBaseFloatingPickerSuggestionProps;
+  pickerSuggestionsProps?: IBaseFloatingPickerSuggestionProps<T>;
+
   /**
    * A callback for when a persona is removed from the suggestion list
    */
-  onRemoveSuggestion?: (item: IPersonaProps) => void;
+  onRemoveSuggestion?: (item: T) => void;
```

Comment:
The way I interpret this PR is that T is guaranteed to extend `IPersonaProps` which would make this backwards compatible. Correct?

Reply:
It doesn't look like `<T>` extends `IPersonaProps` anymore.. this looks like it could be a breaking change now. Did `<T>` change?

Reply:
cc @Adjective-Object

Reply:
My mistake. Sorry, the change is large and I mistook this for a different change.

In this change, onRemoveSuggestion is being cast at all call-sites to be TItem, and was never guaranteed to be an IPersonaProps.

Anyone using onRemoveSuggestion on a non-persona BaseFloatingPicker would have had to be casting at the callsite to get correct behavior